### PR TITLE
PLF-8265 : remove JBoss related modules

### PR DESCRIPTION
**Pull request by thomasdelhomenie**
The JBoss distributions is not supported anymore started from 5.2, so all the modules/resources related to JBoss are removed.

### CODE REVIEW

File: test/servers/wildfly11/dependencies/pom.xml [hunk 1]
```diff
@@ -1,37 +0,0 @@
-
```
Comment (vsellier) [inline test/servers/wildfly11/dependencies/pom.xml]: shouldn't the dependency on this module in the main pom.xml be removed ?
https://github.com/exodev/gatein-pc/blob/fix/PLF-8265/pom.xml#L211

File: test/servers/wildfly11/run/pom.xml [hunk 2]
```diff
@@ -1,101 +0,0 @@
...
-      <!-- Arquillian container -->
-      <dependency>
-          <groupId>org.jboss.spec</groupId>
-          <artifactId>jboss-javaee-7.0</artifactId>
-          <type>pom</type>
-          <scope>test</scope>
-      </dependency>
-      <dependency>
-          <groupId>org.wildfly.arquillian</groupId>
-          <artifactId>wildfly-arquillian-container-managed</artifactId>
```
Comment (vsellier) [inline test/servers/wildfly11/run/pom.xml]: It seems it's not used anywhere else so the dependency can be removed on the main pom.xml
https://github.com/exodev/gatein-pc/blob/fix/PLF-8265/pom.xml#L342

File: test/servers/wildfly11/run/pom.xml [hunk 3]
```diff
@@ -1,101 +0,0 @@
...
-        <groupId>org.exoplatform.gatein.pc</groupId>
-        <artifactId>pc-test-core</artifactId>
-        <type>test-jar</type>
-        <classifier>test-sources</classifier>
-      </dependency>
-
-      <!-- Arquillian container -->
-      <dependency>
-          <groupId>org.jboss.spec</groupId>
-          <artifactId>jboss-javaee-7.0</artifactId>
```
Comment (vsellier) [inline test/servers/wildfly11/run/pom.xml]: It seems it's not used anywhere else so the dependency can be removed on the main pom.xml

File: test/servers/wildfly11/dependencies/pom.xml (hunk 1 above)
Comment (thomasdelhomenie) [reply test/servers/wildfly11/dependencies/pom.xml]: Right. Done.

File: test/servers/wildfly11/run/pom.xml (hunk 2 above)
Comment (thomasdelhomenie) [reply test/servers/wildfly11/run/pom.xml]: Right. Done.

File: test/servers/wildfly11/run/pom.xml (hunk 3 above)
Comment (thomasdelhomenie) [reply test/servers/wildfly11/run/pom.xml]: Right. Done.

**Pull request merged**